### PR TITLE
remove azure from package.yaml

### PR DESCRIPTION
**Pull request by muvaf**
### Description of your changes

Looks like I forgot to remove that.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

Manually

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: package/crossplane.yaml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+  name: provider-influxdb
   annotations:
     meta.crossplane.io/maintainer: Crossplane Maintainers <info@crossplane.io>
-    meta.crossplane.io/source: github.com/crossplane/provider-azure
+    meta.crossplane.io/source: github.com/crossplane-contrib/provider-influxdb
     meta.crossplane.io/license: Apache-2.0
     friendly-name.meta.crossplane.io: Provider InfluxDB
     friendly-group-name.meta.crossplane.io/influxdb.crossplane.io: InfluxDB
-    friendly-kind-name.meta.crossplane.io/organizations.azure.crossplane.io: Organization
+    friendly-kind-name.meta.crossplane.io/organizations.influxdb.crossplane.io: Organization
```
Comment (ulucinar) [inline package/crossplane.yaml:11]: What exactly are these friendly name annotations used for?

File: examples/providerconfig.yaml [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+# Use it in the endpoint string like the following:
+# http://${SVC_NAME}.${SVC_NAMESPACE}.svc.cluster.local:80
 ---
 apiVersion: influxdb.crossplane.io/v1alpha1
 kind: ProviderConfig
 metadata:
   name: default
 spec:
-  endpoint: http://influxdb.crossplane-system.svc.cluster.local:8086
+  endpoint: http://influxdb-influxdb2.default.svc.cluster.local:80
```
Comment (ulucinar) [inline examples/providerconfig.yaml:15]: Who deploys this K8s service?

**Review by ulucinar — APPROVED** [review]

File: package/crossplane.yaml (hunk 1 above)
Comment (muvaf) [reply package/crossplane.yaml:11]: They are standard package annotations that help UIs that need human-friendly names. See https://github.com/crossplane/crossplane/blob/master/design/one-pager-package-annotations.md for more details.

File: examples/providerconfig.yaml (hunk 2 above)
Comment (muvaf) [reply examples/providerconfig.yaml:15]: It's just an example endpoint that you can deploy with instructions given at the top of the file.

File: package/crossplane.yaml (hunk 1 above)
Comment (ulucinar) [reply package/crossplane.yaml:11]: Thanks @muvaf. My understanding is that it's part of a goal to encourage a consistent set of annotations across providers.

File: examples/providerconfig.yaml (hunk 2 above)
Comment (ulucinar) [reply examples/providerconfig.yaml:15]: Okay, then it's a service deployed by the Helm chart `influxdata/influxdb2`, e.g., `SVC_NAME` is `influxdb-influxdb2` and the service namespace, `SVC_NAMESPACE` is `default`.

**Pull request merged**